modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -34,6 +35,119 @@ admin_manual:release_notes.adoc
+Administrators can find more information in the https://doc.owncloud.com/server/latest/admin_manual/configuration/server/occ_command.html#user-commands[ownCloud documentation about occ] and https://doc.owncloud.com/server/10.9/admin_manual/configuration/user/user_management.html#moving-the-user-home[about user management].
+
+=== Updated App Versions
+
+Find below a list of updated apps in comparison with the 10.8 complete bundle. More information on the changes can be found in the respective changelogs on ownCloud Marketplace.
+
+* https://marketplace.owncloud.com/apps/files_mediaviewer[Media Viewer] 1.0.5
+* https://marketplace.owncloud.com/apps/files_pdfviewer[PDF Viewer] 1.0.1
+* https://marketplace.owncloud.com/apps/files_texteditor[Text Editor] 2.4.1
+* https://marketplace.owncloud.com/apps/guests[Guests] 0.9.3
+* https://marketplace.owncloud.com/apps/oauth2[OAuth2] 0.5.2
+* https://marketplace.owncloud.com/apps/openidconnect[OpenID Connect] 2.1.0
+* https://marketplace.owncloud.com/apps/web[Web] 4.6.0
+* https://marketplace.owncloud.com/apps/windows_network_drive[Windows Network Drive] 2.1.0
+* https://marketplace.owncloud.com/apps/wopi[Microsoft Office Online] 1.5.1
+* https://marketplace.owncloud.com/apps/user_ldap[LDAP Integration] 0.16.0
+
+=== Other Notable Changes
+
+* The handling for inaccessible federated shares has been improved to better avoid removing temporarily unavailable shares. https://github.com/owncloud/core/pull/38474[#38474]
+* Group admins can now add users who are in at least one of their administered groups to their other groups via the HTTP Provisioning API (previously this was only possible via the web-based user management). https://github.com/owncloud/core/pull/39013[#39013]
+* Issues with updating apps via the Market app under certain circumstances have been resolved. https://github.com/owncloud/core/pull/39108[#39108]
+* The logo on the login screen will be displayed correctly again, independent of its size. https://github.com/owncloud/core/pull/39129[#39129]
+* Image orientation information from EXIF data will now be processed correctly so that images will be displayed with the correct orientation, e.g., in the Media Viewer. https://github.com/owncloud/core/pull/39140[#39140]
+* The `occ user:report` command has been fixed so that only real user folders will be counted (instead of counting system folders like, e.g., avatars). Additionally, folders outside of the data directory will also be taken into account (e.g., symlinks). https://github.com/owncloud/core/pull/39223[#39223] https://github.com/owncloud/core/pull/39254[#39254]
+* The "Home" icon in the breadcrumb of the Files app has been replaced with the icon and text of the respective view (e.g., All Files, Deleted Files) so that its functionality is more clear to the user. https://github.com/owncloud/core/pull/39367[#39367]
```

Comment:
```suggestion
* The "Home" icon in the breadcrumb of the Files app has been replaced with the icon and text of the respective view (e.g., All Files, Deleted Files) so that its functionality is clearer to the user. https://github.com/owncloud/core/pull/39367[#39367]
```